packs/data/pathfinder-bestiary-3.db/locathah-hunter.json

```diff
@@ -682,12 +683,59 @@
                     "value": 1
                 },
                 "description": {
-                    "value": "<p>The locathah hunter designates a single creature they can see and hear, or one they're Tracking, as their prey. The hunter gains a +2 circumstance bonus to Perception checks to Seek the prey and to Survival checks to Track the prey.</p>\n<p>The first time the locathah hits their designated prey in a round, they deal an additional [[/r {1d8}[precision]]]{1d8 precision damage}. These effects last until the hunter uses Hunt Prey again.</p>"
+                    "value": "<p>The locathah hunter designates a single creature they can see and hear, or one they're Tracking, as their prey. The hunter gains a +2 circumstance bonus to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Seek]{Seek} the prey and to Survival checks to @UUID[Compendium.pf2e.actionspf2e.Track]{Track} the prey.</p>\n<p>The first time the locathah hits their designated prey in a round, they deal an additional [[/r {1d8}[precision]]]{1d8 precision damage}. These effects last until the hunter uses Hunt Prey again.</p>"
                 },
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "label": "PF2E.NPCAbility.LocathahHuntPrey.Checks",
+                        "option": "hunt-prey-checks",
+                        "priority": 50,
+                        "toggleable": true
+                    },
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "hunt-prey-checks",
+                            "action:seek"
+                        ],
+                        "selector": "perception",
+                        "type": "circumstance",
+                        "value": 2
+                    },
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "hunt-prey-checks",
+                            "action:track"
+                        ],
+                        "selector": "survival",
+                        "type": "circumstance",
+                        "value": 2
+                    },
```

Comment:
condense these two REs into one with multiple selectors and an `or` predicate on the action.